app/controllers/calculators_controller.rb

```diff
@@ -11,10 +11,17 @@ def show; end
 
   def calculate; end
 
-  def calculator;
+  def calculator
     # renders calculator.html.slim
   end
-  
+
+  def receive_recomendations
+    return unless user_signed_in?
+
+    current_user.update_attribute(:receive_recomendations,
```

Comment:
What if I want to stop receive recommendations, do I have an ability to disable this functionality?